plugin/build.gradle.kts

```diff
@@ -354,7 +354,7 @@ tasks.named<Jar>("javadocJar").configure { from(fileTree("..").include("docs/*")
 publishing {
   publications {
     create<MavenPublication>("retrofuturagradle") {
-      shadow.component(this)
+      artifact(combinedShadowJar)
```

Comment:
Now that there is no `component` being published at all, there is no provider for pom.xml and gradle .module metadata - have you checked that it looks correct?

Reply:
It *seems* fine, the .plugin pom looks the same, and the pom in the artifact directory is also the same apart from an empty `<dependencies/>` tag now missing. But I could also just readd the old dummy `shadowJar` task if we want to play it safe.